lib/src/scheduler.dart

```diff
@@ -22,21 +22,27 @@ import 'dart:isolate';
 import 'native/realm_core.dart';
 
 final _receivePortFinalizer = Finalizer<RawReceivePort>((p) => p.close());
-final Scheduler scheduler = Scheduler._();
+late final Scheduler scheduler;
```

Comment:
I think we should not do changes in this file except the `logMessage` change

Reply:
Let's discuss this.